introduction.md

```diff
@@ -1,3 +1,3 @@
-Hi This is Madan!
-
-My favorite code school course is HTML & CSS
\ No newline at end of file
+*Ruby
+*CSS
+*Javascript
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?